pkg/activator/handler/handler_test.go

```diff
@@ -287,8 +288,23 @@ func TestActivationHandler(t *testing.T) {
 	}}
 	for _, test := range tests {
 		t.Run(test.label, func(t *testing.T) {
-			rt := getRT(t, test.probeErr, test.probeCode, test.probeResp, test.wantErr, test.wantBody, "test-host", nil)
-
+			var probeResponses []activatortest.FakeResponse
```

Comment:
I don't want to be pita given this is test code, but :) `probeResponses := make([]activatortest.FakeResponse, 0, len(test.probResp))` :-)
Or `probeResponses := make([]activatortest.FakeResponse,  len(test.probResp))` and then indexed loop below, which is even faster.